static/js/enketo/widgets/simprints.js

```diff
@@ -51,22 +32,28 @@ define( function( require, exports, module ) {
 
     Simprintswidget.prototype._init = function() {
         var $el = $( this.element );
-
-        $el.find( 'input' ).attr( 'disabled', true );
-
-        if ( !window.medicmobile_android ) {
-            $el.append( '<p>Simprints is only supported for android app users.</p>' );
-            return;
-        }
-        if ( !window.medicmobile_android.simprints_reg ) {
-            $el.append( '<p>Simprints is not supported on this version of the wrapper app.</p>' );
+        var $input = $el.find( 'input' );
+        $input.hide();
+        var angularServices = angular.element( document.body ).injector();
+        var $translate = angularServices.get( '$translate' );
+        var service = angularServices.get( 'Simprints' );
+
+        if ( !service.enabled() ) {
+            $translate('simprints.disabled').then(function(label) {
+                $el.append( '<p>' + label + '</p>' );
+            });
```

Comment:
I'd be happier with consistent code formatting within a single file.  It's more important to me that we maintain the enketo style for all boilerplate within our custom widgets, so if you're happier with medic-standard formatting within widget bodies and enketo-style for boilerplate then please continue.